app/schema/api/v3/models.rb

```diff
@@ -252,7 +252,10 @@ def prescription_drug
          is_protocol_drug: {type: :boolean},
          is_deleted: {type: :boolean},
          patient_id: {"$ref" => "#/definitions/uuid"},
-         facility_id: {"$ref" => "#/definitions/uuid"}
+         facility_id: {"$ref" => "#/definitions/uuid"},
+         frequency: {type: :string, enum: PrescriptionDrug.frequencies.keys},
+         duration_in_days: {type: :integer},
```

Comment:
Can frequency and duration be nullable? Or are we relying on the client to omit these params from the request for regular PDs? Can we confirm with the android team?